recipes/demo/freertos.yaml

```diff
@@ -1,4 +1,12 @@
+inherit: [make]
+
 depends:
+    # tools
+    -
+        name: devel::make
+        use: [tools]
+        forward: True
+
```

Comment:
This should not be required because the basement::rootrecipe class does already provide the make tool...

Reply:
It was indeed not needed. I don't know why I've added it in the first place. :confused: Sorry for the noise.

Reply:
No problem. :)